modules/claims_api/spec/sidekiq/poa_updater_spec.rb

```diff
@@ -195,9 +195,23 @@ def create_poa
     poa
   end
 
-  def create_mock_lighthouse_service
+  def mock_service_by_flipper
+    if Flipper.enabled? :claims_api_poa_updater_enables_local_bgs
+      create_mock_local_bgs_service
+    else
+      create_mock_bgs_ext_service
+    end
+  end
```

Comment:
Wonʼt we always call the external service, since we arenʼt enabling the Flipper in any tests? Should we add tests with the feature flag on?